index.css

```diff
@@ -38,4 +39,24 @@ main {
 
 p {
     margin: 0;
+}
+
+#wrapper {
+    border: 2px solid red;
+    height: 463px;
```

Comment:
Setting height is almost never great. I am sure there is better way to acheive similar result

Reply:
Solved